js_modules/dagster-ui/packages/ui-core/src/assets/OverdueTag.tsx

```diff
@@ -45,7 +45,7 @@ export const OverdueTag = ({
   policy: Pick<FreshnessPolicy, 'cronSchedule' | 'cronScheduleTimezone' | 'maximumLagMinutes'>;
   assetKey: AssetKeyInput;
 }) => {
-  const {liveData} = useAssetLiveData(assetKey);
+  const {liveData} = useAssetBaseData(assetKey);
```

Comment:
I think that the "overdueness" info can probably move into the stale version of live data if we want!

Reply:
the stale data is what is really expensive according to Daniel (depends on the exact asset) so keeping it in the base is preferable for now.

Reply:
(otherwise this callsite would need to fetch expensive to calculate stale data)